### PR TITLE
Standardize secrets and variables for system test

### PR DESCRIPTION
**Pull request by chezou**
Add test.dig along with ingest.dig with data.py. This change lets workflows use standardized secrets i.e. `td.apiserver` for pytd's 'endpoint' argument, with enforcing to use `td.database` for future system tests.

FYI @toru-takahashi 
After merging this PR, we can enhance deploy workflow if we could set secrets from some web app on heroku.

### CODE REVIEW

**Comment by toru-takahashi** [comment]
1 Question.

Could we use TD_API_ENDPOINT instead of TD_API_SERVER?
(Or both)

td-client-python use TD_API_SERVER. But, other clients usually use TD_API_ENDPOINT.
Ex. https://github.com/treasure-data/digdag/blob/master/digdag-tests/src/test/java/acceptance/td/Secrets.java#L15

**Comment by chezou** [comment]
@toru-takahashi As of pytd 0.8.0, we had an issue on `TD_API_ENDPOINT` with presto-client https://github.com/treasure-data/pytd/pull/51 so that we need to use `TD_API_SERVER` as of now.

**Comment by toru-takahashi** [comment]
OK. Gotcha.

File: machine-learning-box/house-price-prediction/py_scripts/data.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+def table_exists(database, table, client):
+    from tdclient.errors import NotFoundError
+
+    try:
+        client.api_client.table(database, table)
+        return True
+    except NotFoundError:
+        pass
+
+    return False
```
Comment (takuti) [inline machine-learning-box/house-price-prediction/py_scripts/data.py]: 👍 Let's have these functions in `pytd.Client`, just as a note.

**Pull request merged**